201-event-hubs-create-event-hub-and-consumer-group/azuredeploy.json

```diff
@@ -24,7 +24,7 @@
          "type":"string",
          "defaultValue":"2015-08-01",
          "metadata":{  
-            "description":"Service Bus ApiVersion used by the template"
+            "description":"ApiVersion used by the template"
```

Comment:
yes, please - we're trying to catch them as they come in instead of doing all 300+ at once...